listenstore/listenstore/listen.py

```diff
@@ -9,7 +9,7 @@ def __init__(self, user_id=None, timestamp=None, artist_msid=None, album_msid=No
                  recording_msid=None, data=None):
         self.user_id = user_id
         self.timestamp = timestamp
-        self.ts_since_epoch = calendar.timegm(self.timestamp.utctimetuple())
+        self.ts_since_epoch = calendar.timegm(self.timestamp.utctimetuple()) if self.timestamp else None
```

Comment:
- There can be no way a listen having no timestamp. Also the same with raw_data as it contains the json data. (I guess the reason why that condition was there was to make sure it always stays as dict after the datebase query)
- Regarding `playing_now`, the only difference between the normal listen and playing_now is the presence of `listened_at` attribute. As there must be some way to differentiate between the two on the display page so i created a `playing_now:bool`. Creating separate inherited object for playing_now seems to be a nice idea. The only reason i treated playing_now the same as the traditional listen was i wanted to use the existing functions, and not treat it differently as there is no difference actually.